### PR TITLE
Add a section that explains the CSS extensions to authors.

### LINKED ISSUE #313: CR bug: rearrange section 6
https://www.w3.org/Bugs/Public/show_bug.cgi?id=28408

Section 6 only has one subsection, 6.1, and we could therefore hoist everything up one level.

Suggest that the CSS extensions should be a separate top-level section, preceding rendering.

### PR DESCRIPTION
**Pull request by silviapfeiffer**
No substantial changes are necessary to the rendering sections, since the new section is merely informative.
Some small cleanup of the rendering section title and subsectioning is included.

Closes #313 
Closes https://www.w3.org/Bugs/Public/show_bug.cgi?id=28408

### CODE REVIEW

**Comment by silviapfeiffer** [comment]
Hmm, I rebased on the gh-pages branch - not sure why the two commits of Simon snuck in... ignore for now and just provide feedback on my patches.

**Comment by silviapfeiffer** [comment]
Marked as non-substantive for IPR from ash-nazg.

File: index.bs [hunk 1]
```diff
@@ -2014,6 +2014,132 @@ whose cues all have a <a>cue payload</a> that is <a>WebVTT chapter title text</a
...
+
+
+<h3 id=css-extension-cue>The ''::cue'' pseudo-element</h3>
+
+<p>The ''::cue'' pseudo-element describes the content of all cues of a video element.</p>
+
+<pre>
+CSS example:
+
+video.upper ::cue { text-transform: uppercase }
```
Comment (zcorpan) [inline index.bs]: 'text-transform' is not a safelisted property, it does not apply for WebVTT. Maybe use 'color' instead?

File: index.bs [hunk 2]
```diff
@@ -2014,6 +2014,132 @@ whose cues all have a <a>cue payload</a> that is <a>WebVTT chapter title text</a
...
+
+
+<h3 id=css-extension-region>The ''::cue-region'' pseudo-element</h3>
+
+<p>The ''::cue-region'' pseudo-element describes the content of all cue regions of a video element.</p>
+
+<pre>
+CSS example:
+
+video.upper ::cue-region { text-transform: uppercase }
```
Comment (zcorpan) [inline index.bs]: Remove space in selector

File: index.bs [hunk 3]
```diff
@@ -2014,6 +2014,132 @@ whose cues all have a <a>cue payload</a> that is <a>WebVTT chapter title text</a
...
+
+The above rule means "change the letters of all the cues inside all the cue regions of all video elements
+of class 'upper' to uppercase".
+</pre>
+
+<p>The selector ''::cue-region'' does not match any real document element. It does match a pseudo-element
+that conforming user agents will insert into the shadow dom of the video element to render the
```
Comment (zcorpan) [inline index.bs]: Same as earlier about "shadow dom"

File: index.bs [hunk 4]
```diff
@@ -4592,7 +4718,7 @@ object">WebVTT region objects</a> must take their initial values.</p>
 then they must be interpreted as defined in the next section.</p>
 
 
-<h4 id=css-extensions>CSS extensions</h4>
+<h3 id=css-extensions>Processing CSS extensions</h3>
```
Comment (zcorpan) [inline index.bs]: I think this should be a subsection of the "CSS extensions" section above, so the authoring intro and the processing model are together.

**Comment by silviapfeiffer** [comment]
Thanks for the review - would love for you to take this over - I just made it to help.

**Comment by dwsinger** [comment]
> On Oct 18, 2016, at 20:15 , Silvia Pfeiffer <notifications@github.com> wrote:
> 
> Thanks for the review - would love for you to take this over - I just made it to help.

which it did enormously.  Simon, can you fix it up and apply the patch?  Many many thanks for the review and (prospectively) the action…

> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

David Singer
Manager, Software Standards, Apple Inc.

**Comment by silviapfeiffer** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Comment by zcorpan** [comment]
Finally got around to this. Please review the new text.

**Comment by silviapfeiffer** [comment]
looking good, but you need to resolve the conflict. Thanks!

**Comment by zcorpan** [comment]
Thank you! I've cleaned up the branch for landing.

**Comment by silviapfeiffer** [comment]
Thanks!

**Pull request merged**